### DISCUSSION TITLE
Autoupdate bug

### DISCUSSION THREAD

**Discussion by grantallonanytable** [gitextensions/gitextensions#9401] [General] [answered]
Application offered me to install new version, installed this one:

```
 - Git Extensions 3.5.1.12196
- Build 9099a1e76d01dbbd6896d6b026027db0b68fba96

```
When started new build I saw russian UI was reseted to english, in settings i saw "language:russian", but on whole english UI.
Also at language panel russian flag was absent.
W/A is to find translation file at some branch and to copy from github manually. And restarted program.
It helped.
May be someone can find that bug.

**Comment by pmiossec** [comment, marked as answer]
In fact, it's more the normal behavior since the Russian translation has been removed because it is no more in a good state ( no contributions so there were to many string not translated).

**Reply by pmiossec** [reply]
The commit that remove the translations not in good state for v3.5
https://github.com/gitextensions/gitextensions/commit/01a75d63c1db1d5b7e04bd7eca6fc75021e1b5f4

**Reply by RussKie** [reply]
We only ship translations that are at least 95% complete. The Russian translation (along with others) hasn't met the bar.
![image](https://user-images.githubusercontent.com/4403806/126455309-ffb0fe5c-e284-4c9f-8a54-8abd459610c8.png)

**Reply by RussKie** [reply]
You can pull the incomplete translation from https://www.transifex.com/git-extensions/git-extensions/ui-3_5/

**Reply by pmiossec** [reply]
@grantallonanytable And you could contribute also to see it shipped in the next release 😉
